File: week-6/sentimental-cash/cash.py
from cs50 import get_float

# Get positive input
while True:
    try:
        change = get_float("Change: ")
        if change > 0:
            break
    except ValueError:
        pass

# Convert dollars to cents
cents = int(round(change * 100))

# Calculate the number of coins needed
coins = 0
for coin in [25, 10, 5, 1]:
    coins += cents // coin
    cents %= coin

print(coins)

File: week-1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main()
{

    int cents = 0;
    do
    {
        cents = get_int("Money owed: ");
    }
    while (cents < 0);

    int coins = 0;

    coins += cents / 25;
    cents = cents % 25;

    coins += cents / 10;
    cents = cents % 10;

    coins += cents / 5;
    cents = cents % 5;

    coins += cents;

    printf("%i\n", coins);

    return 0;
}
